umbral-pre/Cargo.toml

```diff
@@ -15,9 +15,12 @@ sha2 = { version = "0.9", default-features = false }
 chacha20poly1305 = { version = "0.8", features = ["xchacha20poly1305"] }
 hkdf = { version = "0.11", default-features = false }
 hex = { version = "0.4", default-features = false, features = ["alloc"] }
-serde = { version = "1", default-features = false, optional = true }
+serde = { version = "1", default-features = false, features = ["derive"], optional = true }
 base64 = { version = "0.13", default-features = false, features = ["alloc"] }
 pyo3 = { version = "0.15", optional = true }
+js-sys = { version = "0.3", optional = true }
+wasm-bindgen = {version = "0.2.74", optional = true }
+wee_alloc = { version = "0.4", optional = true }
```

Comment:
Does it need to be here? It's only used in `umbral-pre-wasm`

Reply:
`wee_alloc` can go, but I need to keep others there because I need `#[wasm-bindgen]` and mapping errors with `js_sys::Error`. I feel like this is similar to keeping `pyo3` for `python-bindings` here, no?

Reply:
Ah yes, I agree about the other too, I was asking about wee_alloc specifically